executor/oci/spec_windows.go

```diff
@@ -19,9 +17,6 @@ func generateMountOpts(resolvConf, hostsFile string) ([]oci.SpecOpts, error) {
 func generateSecurityOpts(mode pb.SecurityMode) ([]oci.SpecOpts, error) {
 	if mode == pb.SecurityMode_INSECURE {
 		return nil, errors.New("no support for running in insecure mode on Windows")
-	} else if system.SeccompSupported() && mode == pb.SecurityMode_SANDBOX {
-		// TODO: Can LCOW support seccomp? Does that even make sense?
-		return []oci.SpecOpts{seccomp.WithDefaultProfile()}, nil
```

Comment:
To answer this question; https://github.com/containerd/containerd/blob/v1.4.1/contrib/seccomp/seccomp_default_unsupported.go#L1-L26

```go
// +build !linux
```

and https://github.com/containerd/containerd/blob/v1.4.1/contrib/seccomp/seccomp_default_unsupported.go#L24-L26

```go
func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
	return &specs.LinuxSeccomp{}
}
```

Reply:
This was more of a forward-looking pondering. I know it doesn't work _now_, but I figured if at some point it was implemented, that was the place to hook it up. Although `system.SeccompSupported()` doesn't really provide a way to indicate whether it's LCOW or not, so we'd still need _some_ more plumbing to activate that.

The whole LCOW thing seems to be somewhat up-in-the-air anyway, I was going to ignore it until after I had WCOW working, if even then.